packages/create-webpack-app/src/types.ts

```diff
@@ -0,0 +1,36 @@
+export type {
+  ActionConfig,
+  AddActionConfig,
+  CustomActionConfig,
+  AddManyActionConfig,
+  AppendActionConfig,
+  CustomActionFunction,
+  ModifyActionConfig,
+  PlopCfg,
+  PlopGenerator,
+  NodePlopAPI,
```

Comment:
Any specific reason to export the types from here when these can be imported from "node-plop" directly? Let's avoid such exports

Reply:
Earlier I had a different plan but now I think i can remove this export

Reply:
[367ec7a](https://github.com/webpack/webpack-cli/pull/4249/commits/367ec7a7086821e4fd088956ab930891041328e6)
Check this out made this change in latest pr.